PyDrocsid/converter.py

```diff
@@ -60,3 +61,40 @@ async def convert(self, ctx: Context[Bot], argument: str) -> User | Member:
             return await ctx.bot.fetch_user(user_id)
         except (NotFound, HTTPException):
             raise BadArgument(t.user_not_found)
+
+
+class DurationConverter(Converter[int | None]):
+    """
+    Converter for retrieving minutes from a string containing different time units
+    """
+
+    async def convert(self, ctx: Context[Bot], argument: str) -> int | None:
+        """
+        Extracts information about years, months, weeks, days, hours and minutes from a string
+        and returns the total amount of time in minutes
```

Comment:
```suggestion
        and returns the total amount of time in minutes.
```
Sätze enden mit einem Punkt ;).